Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py

```diff
@@ -284,7 +290,12 @@ def main():
         else:
             raise NotImplementedError(f'{command} is not an existing Generic SQL command')
     except Exception as err:
-        return_error(f'Unexpected error: {str(err)} \nquery: {demisto.args().get("query")} \n{traceback.format_exc()}')
+        if type(err).__name__ == 'OperationalError':
+            return_error("Unexpected error: certificate verify failed, unable to get local issuer certificate. "
+                         "Try selecting 'Trust any certificate' checkbox in the integration configuration.")
+        else:
+            return_error(
+                f'Unexpected error: {str(err)} \nquery: {demisto.args().get("query")} \n{traceback.format_exc()}')
```

Comment:
```suggestion
                f'Unexpected error: {str(err)} \nquery: {demisto.args().get("query")}')
```
the `return_error` method already takes care about it